pages/linux/bmon.md

```diff
@@ -0,0 +1,19 @@
+# bmon
+
+> Monitor bandwidth and capture network related statistics.
+
+- Display all interfaces:
```

Comment:
So this shows a list of interfaces? Or does it capture on them too? Perhaps we could clarify this.

Reply:
Updated for clarity